components/form/Input.js

```diff
@@ -2,8 +2,9 @@ import React, { useState } from 'react';
 import { useFormContext } from 'react-hook-form';
 import PropTypes from 'prop-types';
 import { ShowPasswordIcon, HidePasswordIcon } from '../../assets/svgs/svgs';
+import { Label, ErrorLabel } from '.';
 
-const Input = ({ className, Icon, label, name, placeholder, type }) => {
+const Input = ({ className, Icon, labelClassName, name, placeholder, text, type }) => {
```

Comment:
Lets revert this label, I think you misintepret the last comment on this.
```suggestion
const Input = ({ className, Icon, labelClassName, name, placeholder, label, type }) => {
```

Reply:
yes have fixed that now